tests/pipelines/test_pipelines_visual_question_answering.py

```diff
@@ -109,6 +136,35 @@ def test_large_model_pt(self):
             [[{"score": 0.8799, "answer": "2"}, {"score": 0.296, "answer": "1"}]] * 2,
         )
 
+    @slow
+    @require_torch
+    def test_large_model_pt_blip2(self):
+        vqa_pipeline = pipeline("visual-question-answering", model="Salesforce/blip2-opt-2.7b")
```

Comment:
```suggestion
        vqa_pipeline = pipeline("visual-question-answering", model="Salesforce/blip2-opt-2.7b", model_kwargs={"torch_dtype": torch.float16)
```
Make also sure to cast the input in `torch.float16` inside `_forward` if needed

Reply:
Hi @younesbelkada , thanks for your feedback :)
model_kwargs were updated in test_large_model_pt_blip2, as recommended by you, but i am not sure how to check if casting to `torch.float16 `inside `_forward` is needed, could you please give me some hints about what should I check? Thanks